src/transport/net_ib.cc

```diff
@@ -1368,6 +1386,8 @@ ncclNet_t ncclNetIb = {
   ncclIbInit,
   ncclIbDevices,
   ncclIbGetProperties,
+  ncclIbGetStatus,
+  ncclIbSetStatus,
```

Comment:
Make the name more related to resilience

Reply:
suggest keep this naming since it's more neutral name and common to use later.